README.md

```diff
@@ -123,11 +123,11 @@ gcloud container clusters delete onlineboutique \
     --project=${PROJECT_ID} --zone=${ZONE}
 ```
 
-## Provision the GKE cluster and deploy the Online Boutique apps with Terraform
+## Use Terraform to provision a GKE cluster and deploy Online Boutique
 
-Instructions for using Terraform to replicate the [**Quickstart (GKE)**](#quickstart-gke) steps can be found [here](terraform).
+The [`/terraform` folder](terraform) contains instructions for using [Terraform](https://www.terraform.io/intro) to replicate the steps from [**Quickstart (GKE)**](#quickstart-gke) above.
```

Comment:
**My rationale for this change:**

I like to avoid the use of less informative links such as "click here".
Specifying "/terraform folder" lets the reader know where they'll be going.